packages/blade/src/components/Spinner/BaseSpinner/BaseLoader.tsx

```diff
@@ -0,0 +1,47 @@
+import { Svg, Path } from '../../Icons/_Svg';
+
+type BaseLoaderProps = {
+  size: string;
+  color: string;
+};
+
+const BaseLoader = ({ size, color }: BaseLoaderProps): React.ReactElement => {
```

Comment:
Yes we do actually. The LoadingIcon is the one which I created first and the created this BaseLoader because Icons have fixed size, colors of the icon. And in here we need a base loader which can accept any size or color.

Reply:
Discussed, We'll get  of the BaseSpinner component and only have a Spinner component that accepts size as 'small', 'medium', etc.